Source/FakeItEasy.Specs/ConstructorWithVirtualCallsSpecs.cs

```diff
@@ -106,16 +260,9 @@ public FakedClass()
             {
                 this.StringPropertyValueDuringConstructorCall = this.StringProperty;
                 this.ValueTypePropertyValueDuringConstructorCall = this.ValueTypeProperty;
-                this.InterfacePropertyMethodResultDuringConstructorCall = this.InterfaceProperty.InnerProperty;
 
                 this.StringProperty = "value set in constructor";
                 this.ValueTypeProperty = 123456;
-                this.InterfaceProperty.InnerProperty = "value set in constructor";

```

Comment:
The benefit the test brought appeared to be that `InterfaceProperty`'s behaviour is properly configured by the time it's used during the constructor—it returns a fake `IInterface`. It seemed to me that this is no different than returning the proper values from the other properties, one of which was a value type and one a reference type.